content/20.pharmaceuticals.md

```diff
@@ -699,52 +699,87 @@ Additionally, different studies used different dosages, number of doses, and met
+
+With the ongoing global spread of new SARS-CoV-2 variants, there is a growing concern that mutations in SARS-CoV-2 spike protein can escape antibody neutralization, thereby reducing the efficacy of monoclonal antibody therapeutics and vaccines.
+A comprehensive mutagenesis screen recently identified several amino acid substitutions in the SARS-CoV-2 spike protein that can prevent antibody neutralization [@doi:10.1016/j.chom.2021.01.014].
+While some mutations result in resistance to only one antibody, others confer broad resistance to multiple monoclonal antibodies as well as polyclonal huma sera, suggesting that some amino acids are “hotspots” for antibody resistance.
+However, it was not investigated whether the resistant mutations identified result in a fitness advantage.
+Accordingly, an impact on neutralizing efficiency has been reported for the emerging UK (B.1.1.7) and South Africa (B.1.351) variants [@doi:10.21203/rs.3.rs-228079/v1; @doi:10.1101/2021.02.03.429355; @doi:10.1101/2021.01.25.428137].
+While the reported impact on antibody neutralization needs to be confirmed in vivo, it suggests that some adjustments to therapeutic antibody treatments may be necessary to maintain the efficacy that was reported in previous clinical trials. 
+
+Antibody cocktails such as REGN-COV2, CT-P59, and AZD7442 have been developed to overcome the risk for attenuation of neutralizing activity of a single monoclonal antibody.
+These cocktails consist of antibodies that recognize different epitopes on the spike protein, decreasing the likelihood that a single amino acid change can cause resistance to all antibodies in the cocktail.
+However, neutralizing resistance can emerge even against an antibody cocktail if the individual antibodies target subdominant epitopes [@doi:10.1101/2021.02.03.429355]. 
+
+Another strategy is to develop broadly neutralizing antibodies that target structures that are highly conserved, as these are less likely to mutate [@doi:10.1016/j.immuni.2007.11.018; @doi:10.1126/science.1171491] or to target epitopes that are insensitive to mutations [@doi:10.1016/j.vaccine.2006.04.054].
+One such antibody (ADG-2) has recently been reported [@doi:10.1126/science.abf4830].
+This antibody targets a highly conserved epitope that overlaps the hACE2 receptor binding site of all clade 1 sarbecoviruses.
+Prophylactic administration of ADG-2 in an immunocompetent mouse model of COVID-19 resulted in protection against viral replication in the lungs and respiratory burden.
+Since the epitope targeted by ADG-2 represents an Achilles’heel for clade 1 sarbecoviruses, this antibody might be a promising candidate against all circulating variants as well as emerging SARS-related coronaviruses.
+
+In just under a year since the first structure of the SARS-CoV-2 spike protein has been published, an impressive pipeline of monoclonal antibodies targeting SARS-CoV-2 has entered clinical trials with hundreds of more candidates in preclinical stages.
```

Comment:
```suggestion
In just under a year since the structure of the SARS-CoV-2 spike protein was first published, an impressive pipeline of monoclonal antibodies targeting SARS-CoV-2 has entered clinical trials, with hundreds more candidates in preclinical stages.
```